app/serializers/notification_serializer.rb

```diff
@@ -55,4 +56,12 @@ def acting_user_avatar_template
   def include_acting_user_avatar_template?
     object.acting_user.present?
   end
+
+  def acting_user_name
+    object.acting_user&.name
+  end
+
+  def include_acting_user_full_name?
```

Comment:
```suggestion
  def include_acting_user_name?
```

I think this should be the above because there is no `acting_user_full_name` attribute